backend/clickhouse/log_row.go

```diff
@@ -104,39 +98,35 @@ func WithSeverityText(severityText string) LogRowOption {
 	}
 }
 
-func WithSource(source string) LogRowOption {
-	return func(h *LogRow) {
-		if source == highlight.SourceAttributeFrontend {
-			h.Source = LogRowSourceValueFrontend
-		} else {
-			h.Source = LogRowSourceValueBackend
-		}
+func WithSource(source modelInputs.LogSource) LogRowOption {
```

Comment:
Note that `modelInputs.LogSource` is technically a `string`

https://github.com/highlight/highlight/blob/d8ee3158b7599a8e1fcecde73e3d701e4d84b274/backend/private-graph/graph/model/models_gen.go#L953

So any string can be passed in here. 

Not sure if it's possible for gqlgen to give Go a type that's more strict (see [discussion](https://github.com/99designs/gqlgen/discussions/1898)).